js/customEvents.js

```diff
@@ -13,607 +13,609 @@
+
+                            });
+
+                        } else {
+
+                            addEvent(SVGelem.element, event, function(e) {
+
+                                e.stopPropagation();
+
+                                if (isSeries) { // #108, #93 - references in e.point and this after chart.update()
+                                    var chart = eventObject.chart,
+                                        normalizedEvent = chart.pointer.normalize(e);
+
+                                    if (!eventObject.directTouch) {
+                                        elemObj = eventObject.searchPoint(normalizedEvent, eventObject.kdDimensions === 1);
+                                    }
+
+                                    e.point = elemObj; //	#89 point reference in mouse event
+                                }
+
+                                if ((eventObject && !isPoint) || (eventObject && isNumber(eventObject.value))) { // #95 wrong reference for axis labels
+                                    eventObject.value = elemObj.textStr || eventObject.value; // #95, #98 wrong reference for axis labels
+                                    elemObj = eventObject;
+                                }
+
+                                if (elemObj && elemObj.textStr) { // labels
+                                    elemObj.value = elemObj.textStr;
+                                }
+
+                                if (
+                                    isPoint &&
+                                    event === 'click' &&
+                                    elemObj.series &&
+                                    elemObj.series.options &&
+                                    elemObj.series.options.allowPointSelect
+                                ) {
+                                    var defaultFunction = function(event) {
+                                        // Control key is for Windows, meta (= Cmd key) for Mac, Shift
+                                        // for Opera.
+                                        if (elemObj.select) { // #2911
+                                            elemObj.select(null, e.ctrlKey || e.metaKey || e.shiftKey);
+                                        }
+                                    };
+
+                                    HC.fireEvent(elemObj, event, e, defaultFunction);
+                                }
+
+                                if (elemObj && elemObj.drilldown) { // #114 - drillUp - undefined ddDupes []
+                                    elemObj.doDrilldown(undefined, undefined, e);
+                                }
+                                if (event && events[event]) { // #150 - default highcharts event binding undefined
+                                    events[event].call(elemObj, e);
+                                } else {
+                                    events[event].call(elemObj, e);
+                                }
```

Comment:
Changes ends here

Reply:
@richie50 Hi, just wondering why both if and else branches end up with the same `events[event].call(elemObj, e);`? forgot to change else branch content?